projects/valtimo/plugin/src/lib/plugins/catalogi-api/components/get-eigenschap/get-eigenschap-configuration.component.ts

```diff
@@ -0,0 +1,75 @@
+/*!
+ * Copyright 2015-2024 Ritense BV, the Netherlands.
+ *
+ * Licensed under EUPL, Version 1.2 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://joinup.ec.europa.eu/collection/eupl/eupl-text-eupl-12
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" basis,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {Component, EventEmitter, Input, OnDestroy, OnInit, Output} from '@angular/core';
+import {BehaviorSubject, combineLatest, Observable, Subscription, take} from 'rxjs';
+import {GetEigenschapConfig} from '../../models';
+import {FunctionConfigurationComponent} from '../../../../models';
+
+@Component({
+  selector: 'valtimo-get-eigenschap-configuration',
+  templateUrl: './get-eigenschap-configuration.component.html',
+  styleUrls: ['./get-eigenschap-configuration.component.scss']
+})
+
+export class GetEigenschapConfigurationComponent
+  implements FunctionConfigurationComponent, OnInit, OnDestroy
+{
+  @Input() save$: Observable<void>;
+  @Input() disabled$: Observable<boolean>;
+  @Input() pluginId: string;
+  @Input() prefillConfiguration$: Observable<GetEigenschapConfig>;
+  @Output() valid: EventEmitter<boolean> = new EventEmitter<boolean>();
+  @Output() configuration: EventEmitter<GetEigenschapConfig> =
+    new EventEmitter<GetEigenschapConfig>();
+
+  private saveSubscription!: Subscription;
+
+  private readonly formValue$ = new BehaviorSubject<GetEigenschapConfig | null>(null);
+  private readonly  valid$ = new BehaviorSubject<boolean>(false);
+
+  ngOnInit(): void {
```

Comment:
public. This file does not conform to coding guidelines, please consult https://github.com/valtimo-platform/valtimo-frontend-libraries/blob/next-minor/CODING-GUIDELINES.md